### PR TITLE
Project News Site - Game Zone - Karan Mann

### PR DESCRIPTION
**Pull request by karanmann**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,89 @@
...
+        <h3>Watchdogs 2</h3>
+      </div>
 
-    <section class="">
-      <!-- The other news  -->
     </section>
+
+    <Footer>
+      <p class="Footer">🦊 Game Zone News by Karan Mann 🦊</p>
+    </Footer>
```
Comment (HIPPIEKICK) [inline code/index.html:86]: HTML elements are not case sensitive but it's a convention to write them in lowercase

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,334 @@
...
+  margin-left: 30px;
+  margin-right: 30px;
+}
+.rightlinks:hover {
+  color: blue;
+}
+
+.leftlinks:hover {
+  color: blue;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:46]: And this to:
```
.leftlinks:hover, .rightlinks:hover {
  color: blue;
}
```

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,334 @@
...
+
+.rightlinks {
+  display: none;
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  margin-left: 30px;
+  margin-right: 30px;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:39]: Since these have the same styling, you can shorten this to:
```
.leftlinks, .rightlinks {
  display: none;
  text-decoration: none;
  color: black;
  font-weight: bold;
  margin-left: 30px;
  margin-right: 30px;
}
```

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,334 @@
...
+  color: blue;
+}
+
+.logo {
+  height: 50px;
+  margin: 10px 20px;
+}
+nav {
+  display: flex;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:54]: Keep all the nav styling in one CSS declaration (you already have one `nav` styling section)

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,334 @@
...
+
+nav a {
+  color: #575555;
+  padding: 23px 30px;
+  text-decoration: none;
+  font-size: 14px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  letter-spacing: 3px;
+  align-self: flex;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:64]: This is the same as `.leftlinks` and `.rightlinks`, right?

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,334 @@
...
+
+nav a:hover {
+  color: grey;
+  background-color: lightgrey;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:69]: And this too (`.leftlinks: hover, .rightlinks:hover`)

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,334 @@
...
+
+footer {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0px;
+  background-color: #f6f4e6;
+  width: 100%;
+  height: 125px;
+}
```
Comment (HIPPIEKICK) [inline code/style.css:132]: Nice footer!

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,334 @@
...
+
+@media (max-width: 666) {
+  .leftlinks {
+    display: none;
+  }
+
+  .rightlinks {
+    display: none;
+  }
+}
```
Comment (HIPPIEKICK) [inline code/style.css:182]: If you code mobile first, you don't need a media query for mobile since it's default :)

File: code/style.css [hunk 9]
```diff
@@ -1,4 +1,334 @@
...
+    letter-spacing: 1px;
+    padding-right: 2px;
+    text-decoration: none;
+  }
+  .rightlinks {
+    display: inline-flex;
+    letter-spacing: 1px;
+    padding-left: 2px;
+    text-decoration: none;
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:221]: This could also be shortened:
```
  .leftlinks, .rightlinks {
    display: inline-flex;
    letter-spacing: 1px;
    padding-right: 2px;
    text-decoration: none;
  }
```

File: code/style.css [hunk 10]
```diff
@@ -1,4 +1,334 @@
...
+  align-content: center;
+}
+
+.cards {
+  background-color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  flex-basis: auto;
+  margin: 20px 0;
```
Comment (HIPPIEKICK) [inline code/style.css:151]: Consider changing this to `margin: auto;` to center the cards

File: code/style.css [hunk 11]
```diff
@@ -1,4 +1,334 @@
...
+footer {
+  background-color: black;
+  color: white;
+  text-align: center;
+}
+
+/* Desktop */
+@media (min-width: 1025px) {
+  .leftlinks,
+  .rightlinks {
```
Comment (HIPPIEKICK) [inline code/style.css:272]: Yeah, that's right ;)

File: code/style.css [hunk 12]
```diff
@@ -1,4 +1,334 @@
...
+    transition: box-shadow 0.3s ease;
+  }
+
+  .cards {
+    width: 22%;
+  }
+
+  .cards:hover {
+    box-shadow: 5px 10px 10px #999;
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:326]: This looks super!

File: code/index.html [hunk 13]
```diff
@@ -1,22 +1,89 @@
...
+      </section>
+    </main>
+
+    <hr/>
+
+    <h3 class="latestnews.">GAMES YOU SHOULD PLAY ...</h3>
+
+    <section class="newscards">  <!-- NEWS CARDS-->
+     
+      <div class="cards">
```
Comment (HIPPIEKICK) [inline code/index.html:47]: Think about creating clear class names for your elements. This is a singular card

**Review by HIPPIEKICK — APPROVED** [review]
Good job with the responsive design Karan! I really like your box shadows and hover effects 👍 Think about DRY (don't repeat yourself) in the CSS-file and consider getting rid of some line breaks in the HTML. Apart from that - keep up the good work!

**Comment by karanmann** [comment]
Hej Matilda, 

I am now aware of a lot of mistakes that I was making since my basics on my concepts were not clear so I was making a lot of repetitions. But now that I know better how to apply them I would be using your recommendation. 

Thank you again for your invaluable feedback.

This makes me a bit more confident as I have basically 0 coding experience.
